core/src/main/java/org/testcontainers/utility/TestcontainersConfiguration.java

```diff
@@ -24,7 +24,8 @@
 
     private String ambassadorContainerImage = "richnorth/ambassador:latest";
     private String vncRecordedContainerImage = "richnorth/vnc-recorder:latest";
-    private String tinyImage = "alpine:3.2";
+    private String tinyImage = "alpine:3.5";
+    private Boolean disableChecks = false;
```

Comment:
leftover from previous implementation actually :) will change to boolean, thanks :)